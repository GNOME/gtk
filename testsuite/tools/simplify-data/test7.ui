<!-- default layout property gets stripped -->
<interface>
  <object class="GtkGrid">
    <child>
      <object class="GtkLabel">
        <layout>
          <property name="column-span">1</property>
          <property name="row-span">2</property>
        </layout>
      </object>
    </child>
  </object>
</interface>
