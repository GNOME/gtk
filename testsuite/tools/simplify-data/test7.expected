<?xml version="1.0" encoding="UTF-8"?>
<interface>
  <object class="GtkGrid">
    <child>
      <object class="GtkLabel">
        <layout>
          <property name="row-span">2</property>
        </layout>
      </object>
    </child>
  </object>
</interface>
